core/src/main/thrift/manager.thrift

```diff
@@ -213,7 +213,7 @@ service FateService {
 
 }
 
-service MasterClientService extends FateService {
```

Comment:
Will renaming this will break client-server compatibility between 2.0 and 2.1, if it is not already broken?

I'm okay if we draw a line in the sand and say that 2.0 clients can't talk to 2.1, but I thought that there was some previous decisions made to try to preserve that to some extent, and I can't recall the details.

Reply:
Ah, interesting. I thought you said in a previous PR that wire compatibility was already broken between 2.0 and 2.1. If it's not, then yeah, we might want to reconsider.

Reply:
Wire compatibility is definitely broken between 2.0 and 2.1 for server-to-server communication (preventing rolling upgrades, for example), but I don't know that we have any explicit check to determine client-to-server wire compatibility. So, it may or may not be broken. I'm not sure.

I'm okay with breaking it... if we document that in the release notes. I just want to ensure we are noticing the consequences and accepting the risks as we go.

Reply:
Hmm, I'm not sure. I think if it's not already broken, we're better off not doing any changes to the thrift package in this release (since we already chose to hold the package renaming to support balancer compatibility). I'll do a sniff test with the 2.0.1 client against a 2.1 (main branch) snapshot and see if I can uncover any issues.

Reply:
I spun up a cluster from a tarball built from the main branch, and then ran the accumulo shell from a 2.0.1 install (but client properties point to my 2.1 snapshot instance), and while some things worked in the shell, scanning a table hung indefinitely (but worked from the 2.1 shell), so I'd say client compatibility was already broken. Given that, it would seem the best approach is just to document that fact in the release notes.